modules/ivc_champva/app/form_mappings/vha_10_10d.json.erb

```diff
@@ -1,7 +1,7 @@
 {
   "form1[0].#subform[0].VeteransLastName[0]":            "<%= form.data.dig('veteran', 'full_name', 'last') %>",
   "form1[0].#subform[0].VeteransFirstName[0]":           "<%= form.data.dig('veteran', 'full_name', 'first') %>",
-  "form1[0].#subform[0].VeteransMI[0]":                  "<%= form.data.dig('veteran', 'full_name', 'middle') %>",
+  "form1[0].#subform[0].VeteransMI[0]":                  "<%= form.data.dig('veteran', 'full_name', 'middle')&.first %>",
```

Comment:
Grabs the first letter from middle name to verify we're only printing middle initial on PDF